trans_documentation/impressum.dita

```diff
@@ -40,8 +40,8 @@
     <section>
       <title>Legal Notice on Copyright</title>
       <p>The guidelines are protected by copyright. <xref
-          href="https://creativecommons.org/licenses/by-sa/3.0/de/" format="html" scope="external"
-          >Creative Commons: Namensnennung - Weitergabe unter gleichen Bedingungen 3.0 Deutschland
+          href="https://creativecommons.org/licenses/by-sa/3.0/de/deed.en" format="html" scope="external"
+          >Creative Commons: Attribution - ShareAlike 3.0 Germany
```

Comment:
Why ShareAlike? Don't we want broad reuse, even in commercial/proprietary settings?